core/templates/domain/opportunity/exploration-opportunity-summary.model.spec.ts

```diff
@@ -83,6 +86,56 @@ describe('Exploration opportunity summary model', () => {
         .toEqual(15);
     });
 
+    it('should return a correct translations in review counts', () => {
```

Comment:
```suggestion
    it('should return correct translations in review counts', () => {
```

Omit "a". Here and below.

Reply:
Done